NetCord/Application.cs

```diff
@@ -65,13 +44,177 @@ public Application(JsonModels.JsonApplication jsonModel, RestClient client) : ba
             IntegrationTypesConfiguration = integrationTypesConfiguration.ToDictionary(i => i.Key, i => new ApplicationIntegrationTypeConfiguration(i.Value));
     }
 
+    /// <summary>
+    /// The ID of the application.
+    /// </summary>
+    public override ulong Id => _jsonModel.Id;
+
+    /// <summary>
+    /// The name of the application.
+    /// </summary>
+    public string Name => _jsonModel.Name;
+
+    /// <summary>
+    /// The icon hash of the application.
+    /// </summary>
+    public string? IconHash => _jsonModel.IconHash;
+
+    /// <summary>
+    /// The description of the application.
+    /// </summary>
+    public string Description => _jsonModel.Description;
+
+    /// <summary>
+    /// The application's RPC origin URL list, where <c>T</c> is of type <see cref="string"/>. <see langword="null"/> if RPC is disabled.
```

Comment:
The `<c>T</c> is of type <see cref="string"/>` seems a bit odd. I don't think we do something like that elsewhere.